deeprvat/data/dense_gt.py

```diff
@@ -256,9 +284,19 @@ def setup_phenotypes(
         sim_phenotype_file: Optional[str],
         skip_y_na: bool,
         skip_x_na: bool,
+        sample_file: Optional[str],
     ):
         logger.debug("Reading phenotype dataframe")
         self.phenotype_df = pd.read_parquet(phenotype_file, engine="pyarrow")
+        gt_file = h5py.File(
```

Comment:
Should use a `with` statement here so that the file is closed if there's an exception

Reply:
updated this